storage/innobase/xtrabackup/test/inc/keyring_kms.sh

```diff
@@ -51,16 +51,16 @@ function cleanup_keyring() {
 
 function ping_kms()
 {
-  if [ -z ${KMS_REGION+x} ]; then
+  if [ -z ${KMS_REGION} ]; then
```

Comment:
is it to report failure when KMS_REGION is set to null? Also better to add quotes.

Reply:
Yes, this is to skip the test if we have not SET the variables.